app/api/event_copy.py

```diff
@@ -45,13 +46,23 @@ def create_event_copy(identifier):
     discount_codes = DiscountCode.query.filter_by(event_id=event.id).all()
     speaker_calls = SpeakersCall.query.filter_by(event_id=event.id).all()
     user_event_roles = UsersEventsRoles.query.filter_by(event_id=event.id).all()
+    tax = Tax.query.filter_by(event_id=event.id).all()
 
     db.session.expunge(event)  # expunge the object from session
     make_transient(event)
     delattr(event, 'id')
     event.identifier = get_new_event_identifier()
     save_to_db(event)
 
+    # Ensure tax information is copied
+    if len(tax) > 0:
```

Comment:
@uds5501 can't we use a for loop like other cases here and do the following operation?

Reply:
@shreyanshdwivedi thinking about it, I guess you are right, I will amend this.